test.py

```diff
@@ -87,8 +88,8 @@ def test_build(args, expected=None, unexpected=None):
 check_call('./emsdk update-tags')
 
 print('test latest-releases-upstream')
-check_call('python2 ./emsdk install latest-upstream')
-check_call('./emsdk activate latest-upstream')
+check_call('python2 ./emsdk.py install latest-upstream')
+check_call('python3 ./emsdk.py activate latest-upstream')
```

Comment:
In addition I'd keep the old check `check_call('./emsdk activate latest-upstream')`

Reply:
Reverted